includes/class-amp-travel-taxonomies.php

```diff
@@ -22,6 +29,108 @@ public function init() {
 		add_action( 'activity_edit_form_fields', array( $this, 'edit_activity_meta_fields' ) );
 		add_action( 'edit_activity', array( $this, 'save_activity_meta' ) );
 		add_action( 'create_activity', array( $this, 'save_activity_meta' ) );
+
+		add_action( 'location_add_form_fields', array( $this, 'add_location_meta_fields' ) );
+		add_action( 'location_edit_form_fields', array( $this, 'edit_location_meta_fields' ) );
+		add_action( 'edit_location', array( $this, 'save_location_meta' ) );
+		add_action( 'create_location', array( $this, 'save_location_meta' ) );
+
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_admin_scripts' ) );
+	}
+
+	/**
+	 * Enqueue admin scripts.
+	 */
+	public function enqueue_admin_scripts() {
+		if ( ! isset( $_GET['taxonomy'] ) || 'location' !== $_GET['taxonomy'] ) {
+			return;
+		}
+		wp_enqueue_media();
+
+		wp_enqueue_script(
+			'amp-travel-term-controls',
+			get_template_directory_uri() . '/assets/js/term-controls.js',
+			array( 'underscore', 'jquery' )
+		);
+	}
+
+	/**
+	 * Add location meta fields to add form.
+	 */
+	public function add_location_meta_fields() {
+		?>
+		<div class='form-field form-required location-cover-image-wrap'>
+			<label for='location-cover-image'><?php esc_html_e( 'Activity Cover Image' ); ?></label>
+			<input type='hidden' id='location-cover-image-value' class='small-text' name='location-cover-image-value' value='' />
+			<input type='button' id='location-cover-image' class='button location-cover-image-upload-button' value='<?php esc_html_e( 'Upload' ); ?>' />
+			<input type='button' id='location-cover-image-remove' class='button location-cover-image-upload-button-remove' value='<?php esc_html_e( 'Remove' ); ?>' />
+			<div id='location-cover-image-preview'></div>
+		</div>
+		<div class='form-field form-required location-is-featured-wrape'>
+			<label for='location-is-featured'><?php esc_html_e( 'Featured destination' ); ?></label>
+			<input type="checkbox" name="location-featured-destination" value="1" />
+		</div>
+		<?php
+	}
+
+	/**
+	 * Add meta fields to term edit.
+	 *
+	 * @param WP_Term $term WP_Term being edited.
+	 */
+	public function edit_location_meta_fields( $term ) {
+		$cover_img_id = get_term_meta( $term->term_id, 'amp_travel_location_img', true );
+		if ( empty( $cover_img_id ) ) {
+			$cover_img_id = '';
+		} else {
+			$img_src = wp_get_attachment_image_src( $cover_img_id );
+		}
+
+		$is_featured = get_term_meta( $term->term_id, 'amp_travel_featured', true );
+
+		?>
+		<tr class='form-field form-required location-cover-image-wrap'>
+			<th scope="row"><label for='location-cover-image'><?php esc_html_e( 'Activity Cover Image' ); ?></label></th>
+			<td>
+				<?php wp_nonce_field( basename( __FILE__ ), 'travel_location_meta_nonce' ); ?>
+
+				<input type='hidden' id='location-cover-image-value' class='small-text' name='location-cover-image-value' value='<?php esc_attr( $cover_img_id ); ?>' />
```

Comment:
```php
<?php esc_attr( $cover_img_id ); ?>
```
This could probably use an `echo` at the beginning:
```php
<?php echo esc_attr( $cover_img_id ); ?>
```